### ISSUE TITLE
no setup.py and --user-feature param

### ISSUE THREAD

**Issue by it-can-bee** [microsoft/Graphormer#174]
Hello, follow the provided readme to run the bash script all the way, at the end of pip install. --use-feature=in-tree-build

python setup.py build_ext --inplace error no such option: --use-feature

(null): can't open file 'setup.py': [Errno 2] No such file or directory

, So I try cd cd fairseq directory and git submodule update --init --recursive submodule installation after pip install. --use-feature=in-tree-build. no such option: --use-feature

In other words, the source code of the project may not be fully uploaded, and the setup.py and --use-feature parameters are missing

**Comment by tkreiman** [comment]
I think this can be fixed with:

python -m pip install --upgrade "pip==21.1"

**Comment by javierpey** [comment]
Make sure you run "git submodule update --init --recursive" as per what the script mentioned in install.sh.
